src/openlifu/db/database.py

```diff
@@ -478,6 +534,19 @@ def get_volume_info(self, subject_id, volume_id):
                     "name": volume["name"],\
                     "data_abspath": Path(volume_metadata_filepath).parent/volume["data_filename"]}
 
+    def get_photoscan_info(self, subject_id, session_id, photoscan_id):
+        photoscan_metadata_filepath = self.get_photoscan_metadata_filepath(subject_id, session_id, photoscan_id)
+        with open(photoscan_metadata_filepath) as f:
+            photoscan = json.load(f)
```

Comment:
```suggestion
        photoscan = json.loads(photoscan_metadata_filepath.read_text())
```
(unimportant style tip)
[Path.read_text](https://docs.python.org/3/library/pathlib.html#pathlib.Path.read_text) is super handy and this saves a level of indentation